test/integration/gateway_test.go

```diff
@@ -80,14 +78,15 @@ func TestGatewayEssentials(t *testing.T) {
 	require.Len(t, controlplanes, 1)
 	controlplane := controlplanes[0]
 
-	t.Log("verifying networkpolicies are created")
-	require.Eventually(t, testutils.GatewayNetworkPoliciesExist(t, GetCtx(), gateway, clients), testutils.SubresourceReadinessWait, time.Second)
+	// NOTE: We're not verifying if the NetworkPolicies are created
+	// in integration tests.
+	// Code ref: https://github.com/Kong/kong-operator/blob/27e3c46cd201bf3d03d2e81000239b047da2b2ce/controller/gateway/controller.go#L397-L410
 
 	t.Log("verifying connectivity to the Gateway")
 	require.Eventually(t, Expect404WithNoRouteFunc(t, GetCtx(), "http://"+gatewayIPAddress), testutils.SubresourceReadinessWait, time.Second)
 
 	t.Log("verifying GatewayClass has supportedFeatures set")
-	requiredFeatures, err := gatewayapi.GetSupportedFeatures(consts.RouterFlavorExpressions)
+	requiredFeatures, err := gatewayapi.GetSupportedFeatures(consts.RouterFlavorTraditionalCompatible)
```

Comment:
```suggestion
	requiredFeatures, err := gatewayapi.GetSupportedFeatures(consts.DefaultRouterFlavor)
```

I think it would be more convenient to use the DefaultRouterFlavor value directly here.